### PR TITLE
some suggestions

### PR DESCRIPTION
**Pull request by SebRollen**
Some suggestions for your script! Will leave some inline comments explaining some of the changes

### CODE REVIEW

File: compiled_julia_test.jl [hunk 1]
```diff
@@ -1,7 +1,6 @@
 using PackageCompiler
 using OhMyREPL
 using Plots
-using Statistics
```
Comment (SebRollen) [inline compiled_julia_test.jl:4]: `Statistics` is already a standard library and so is already precompiled - no benefit in precompiling it here

File: julia_sysimage_precompile.zsh [hunk 2]
```diff
@@ -6,11 +6,12 @@ export PATH
 # If needed, testore default ahead of time to not re-use compiled version
 # julia -e 'using PackageCompiler; PackageCompiler.restore_default_sysimage();'
 
-cd ~/src/julia_precompile
+pushd ~/src/julia_precompile
```
Comment (SebRollen) [inline julia_sysimage_precompile.zsh:9]: `pushd` and `popd` allow you to get back to your original directory, so it's just a QOL improvement for if the script is run interactively

File: julia_sysimage_precompile.zsh [hunk 3]
```diff
@@ -6,11 +6,9 @@ export PATH
...
 
-cd ~/src/julia_precompile
-
-# Define the image
-julia --trace-compile=julia_sysimage_precompile_data.jl julia_sysimage_precompile_prep.jl 
+pushd ~/src/julia_precompile
 
 # Compile the image
-julia -e 'using PackageCompiler; PackageCompiler.create_sysimage([:Plots, :OhMyREPL, :DataFrames, :DataFramesMeta, :Chain, :Statistics]; sysimage_path="julia_plus.so", precompile_statements_file="julia_sysimage_precompile_data.jl");'
+julia -e 'using PackageCompiler; PackageCompiler.create_sysimage([:Plots, :OhMyREPL, :DataFrames, :DataFramesMeta, :Chain, :Statistics]; sysimage_path="julia_plus.so", precompile_execution_file="compiled_julia_test.jl");'
```
Comment (SebRollen) [inline julia_sysimage_precompile.zsh:12]: using `precompile_execution_file` here lets you skip the trace-compile line above

**Comment by bestdan** [comment]
Thanks! 

LGTM!

**Pull request merged**